src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Collections/IntervalTree`1.cs

```diff
@@ -226,21 +134,10 @@ private bool Any<TIntrospector>(int start, int length, TestInterval<TIntrospecto
         return false;
     }
 
-    private ImmutableArray<T> GetIntervalsThatMatch<TIntrospector>(
-        int start, int length, TestInterval<TIntrospector> testInterval, in TIntrospector introspector)
-        where TIntrospector : struct, IIntervalIntrospector<T>
-    {
-        using var result = TemporaryArray<T>.Empty;
-        FillWithIntervalsThatMatch(start, length, testInterval, ref result.AsRef(), in introspector, stopAfterFirst: false);
-        return result.ToImmutableAndClear();
-    }
-
-    /// <returns>The number of matching intervals found by the method.</returns>
-    private int FillWithIntervalsThatMatch<TIntrospector>(
-        int start, int length, TestInterval<TIntrospector> testInterval,
+    int IIntervalTree<T>.FillWithIntervalsThatMatch<TIntrospector>(
+        int start, int length, TestInterval<T, TIntrospector> testInterval,
```

Comment:
switched to explicit impl as nothing should call this directly, just the algorithms type.